controllers/machinehealthcheck_controller.go

```diff
@@ -256,12 +258,63 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	for _, t := range unhealthy {
 		condition := conditions.Get(t.Machine, clusterv1.MachineHealthCheckSuccededCondition)
 
-		if annotations.IsPaused(cluster, t.Machine) {
-			logger.Info("Machine has failed health check, but machine is paused so skipping remediation", "target", t.string(), "reason", condition.Reason, "message", condition.Message)
+		if m.Spec.RemediationTemplate != nil {
+			// If external remediation request already exists,
+			// return early
+			if r.externalRemediationRequestExists(ctx, m, t.Machine.Name) {
+				logger.V(2).Info("External Remediation Request already exists", "machine", t.Machine.Name, "templateGVK", m.Spec.RemediationTemplate.GroupVersionKind())
+				return ctrl.Result{}, nil
+			}
+
+			cloneOwnerRef := &metav1.OwnerReference{
+				APIVersion: clusterv1.GroupVersion.String(),
+				Kind:       t.Machine.Kind,
+				Name:       t.Machine.Name,
+				UID:        t.Machine.UID,
+			}
+
+			from, err := external.Get(ctx, r.Client, m.Spec.RemediationTemplate, t.Machine.Namespace)
+			if err != nil {
+				conditions.MarkFalse(m, clusterv1.ExternalRemediationTemplateAvailable, clusterv1.ExternalRemediationTemplateNotFound, clusterv1.ConditionSeverityError, "External Remediation Template not found")
```

Comment:
Should we include `err.Error()` in the message (or maybe just use it instead of "External Remediation Template not found")?

Reply:
I think we could keep this as it is?